### PR TITLE
fixes #46 Always exit 0 when remove image with force option

### LINKED ISSUE #46: docker rmi -f should always exit 0
I used 'docker rmi -f' in a script and expected it to behave like rm 's -f: If the container (or file) does not exist, still exit cleanly. For another example, mkdir -p is handy in shell scripts to idempotently make a directory tree whether or not it already exists, avoiding additional code to handle such cases.

For example:

```sh
$ docker rmi -f non-existant-image
Error response from daemon: No such image: non-existant-image:latest
$ echo $?
1
$ rm -f non-existant-file
$ echo $?
0
```

I'm not sure about the scope of this request (perhaps it also applies to docker rm), or the validity, but I did hear at Dockercon that you were interested in improving the developer experience no matter how small - and this would improve mine a little bit. Otherwise, I need to run "docker rmi -f imagename || true", or wrap it in a second docker call to make it conditional.

### PR DESCRIPTION
**Pull request by e11137**
Always exit with 0 if the -f force option is used

### CODE REVIEW

**Comment by dgageot** [comment]
Hi @e11137, you have to sign your commit for Gordon the Turtle to like you!

File: cli/command/image/testdata/remove-command-success.Image Deleted with force option.golden [hunk 1]
```diff
@@ -0,0 +1,2 @@
+NotFound:image1
```
Comment (thaJeztah) [inline cli/command/image/testdata/remove-command-success.Image Deleted with force option.golden]: Was discussing with @vdemeester that the naming of these files is a bit odd; perhaps we should normalise them to be lowercase and have spaces replaced with `-`. Given that these files are generated, and this is in line with other files in this tests, we can keep it for now 👍

**Comment by thaJeztah** [comment]
@dgageot it was already fixed

screen shot 2017-06-13 at 13 43 32

**Comment by thaJeztah** [comment]
Thank you @e11137 !

**Pull request merged**